src/main/java/org/example/commands/AddDependencyCommand.java

```diff
@@ -17,6 +18,10 @@ public void execute(Map<String, Node> nodeDependencies) {
     String parentId = scanner.nextLine();
     String childId = scanner.nextLine();
 
+    // Check for possibility of any cycle
+    CyclicDependencyValidator cyclicDependencyValidator = new CyclicDependencyValidator();
```

Comment:
this variable can be class member right?

Reply:
made it class member